eventhubs/spring-cloud-azure-starter-integration-eventhubs/eventhubs-integration/README.md

```diff
@@ -133,7 +133,13 @@ source ./terraform/setup_env.sh
 #### Run with Powershell
 
 ```shell
-. terraform\setup_env.ps1
+terraform\setup_env.ps1
+```
+
+#### Run with Command Prompt or Native Tools Command Prompt
+
+```shell
+terraform\setup_env.bat
```

Comment:
do we need this

Reply:
It's required to set the env in `x64 Native Tools Command Prompt`, it's an enhancement tool for Windows Command Prompt.